runbot_language/model/runbot_language.py

```diff
@@ -0,0 +1,103 @@
+# -*- encoding: utf-8 -*-
```

Comment:
Could you [separate the model files by model name](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming).

* `model/runbot_repo.py`
* `model/runbot_Build.py`